src/main/java/frc/robot/constants/RobotMap.java

```diff
@@ -6,30 +6,24 @@
  * the wiring easier and significantly reduces the number of magic numbers
  * floating around.
  */
-public abstract class RobotMap {
-    public CAN can = new CAN();
-    public PCM pcm = new PCM();
-    public DIO dio = new DIO();
-    public PWM pwm = new PWM();
```

Comment:
i'm pretty sure even if the class is static you still need to instantiate the classes

Reply:
nope